src/EditorFeatures/Core/Options/LegacyGlobalOptionsWorkspaceService.cs

```diff
@@ -20,7 +18,6 @@ namespace Microsoft.CodeAnalysis.Options
     internal sealed class LegacyGlobalOptionsWorkspaceService : ILegacyGlobalOptionsWorkspaceService
```

Comment:
📝 This needs to be updated to use ServiceLayer.Editor

Reply:
Does it? This implements a different service interface than what I added to workspace

Reply:
I missed that it was a different interface, but yes since this is the editor layer. It would be reasonable to add a default implementation in the workspaces layer (even if it just takes no action) and we wouldn't want it to conflict in that case.

Reply:
I can follow up on that when I get back.